### PR TITLE
test upgrades from 0.27.7 instead of 0.27.0

### PR DESCRIPTION
**Pull request by phoracek**
**What this PR does / why we need it**:

Use the last z-stream release of that branch to test the real upgrade
flow. Somebody can upgrade from any of the z-stream releases, but in
order to save resources, we will test only the latest one.

Signed-off-by: Petr Horacek <phoracek@redhat.com>

**Special notes for your reviewer**:

**Release note**:

```release-note
Test upgrades from 0.27.7 z-stream instead of 0.27.0
```

### CODE REVIEW

File: automation/check-patch.e2e-lifecycle-k8s.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ main() {
         export RELEASES_DESELECTOR="${to_be_released}"
     else
         # Don't run all upgrade tests in regular PRs, stick to those released under HCO
-        export RELEASES_SELECTOR="{0.18.0,0.23.0,0.27.0,99.0.0}"
+        export RELEASES_SELECTOR="{0.18.0,0.23.0,0.27.7,99.0.0}"
```
Comment (oshoval) [inline automation/check-patch.e2e-lifecycle-k8s.sh:30]: are spaces allowed here between each release tag ?

File: manifests/cluster-network-addons/0.27.7/cluster-network-addons-operator.0.27.7.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            "imagePullPolicy": "IfNotPresent"
+          }
+        }
+      ]
+    description: Additional networking components for Kubernetes
+spec:
+  displayName: Cluster Network Addons
+  description: Deploy additional networking components for Kubernetes
+  keywords:
+    - Networking
```
Comment (oshoval) [inline manifests/cluster-network-addons/0.27.7/cluster-network-addons-operator.0.27.7.clusterserviceversion.yaml:35]: just making sure, no need here mac v tap or other something that was dropped?

File: automation/check-patch.e2e-lifecycle-k8s.sh (hunk 1 above)
Comment (phoracek) [reply automation/check-patch.e2e-lifecycle-k8s.sh:30]: Not in the current implementation AFAIK.

File: manifests/cluster-network-addons/0.27.7/network-addons-config-example.cr.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1alpha1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  imagePullPolicy: IfNotPresent
+  kubeMacPool: {}
```
Comment (oshoval) [inline manifests/cluster-network-addons/0.27.7/network-addons-config-example.cr.yaml:8]: aint there was mac v tap around ? or on other version / i am wrong

File: manifests/cluster-network-addons/0.27.7/cluster-network-addons-operator.0.27.7.clusterserviceversion.yaml (hunk 2 above)
Comment (phoracek) [reply manifests/cluster-network-addons/0.27.7/cluster-network-addons-operator.0.27.7.clusterserviceversion.yaml:35]: Yeah, it should be updated. Not a part of this PR.

File: manifests/cluster-network-addons/0.27.7/network-addons-config-example.cr.yaml (hunk 3 above)
Comment (phoracek) [reply manifests/cluster-network-addons/0.27.7/network-addons-config-example.cr.yaml:8]: This is version 0.27.7.  The was not macvtap back then.

**Pull request merged**